chart/scripts/post-delete-hook.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+succeeded=()
+failed=()
+echo "Uninstalling Rancher resources in the following namespaces:" "$namespaces"
```

Comment:
```suggestion
echo "Uninstalling Rancher resources in the following namespaces: ${namespaces}"
```

Reply:
Can you change all env var references to `${ENVVAR}` instead of `$ENVVAR`? 

The bracket notation is less ambiguous and imo is generally better practice in bash scripts since you don't encounter issues like `${foo}bar` v.s. `$foobar`.

Reply:
fixed all occurrences